src/views/swap/steps/inputInvoice.js

```diff
@@ -56,9 +56,9 @@ class StyledInputInvoice extends React.Component {
   }
 
   onChange = input => {
-    const valid = input.slice(0, 2);
+    const prefix = input.slice(0, 2);
 
-    if (valid === 'ln') {
+    if (prefix === 'ln' || prefix === 'li') {
```

Comment:
@michael1011 Why dony we check for `ln` or 'lighting`?

Reply:
We could do:
```
if (input.slice(0, 2) === 'ln' || input.slice(0, 10) === 'lightning:') {
```
but I figured that it is fine this way. Do you think that we should do more strict checks in the frontend?

Reply:
Yeah, I think we should it will minimize the number of unnecessary calls to the backend.